src/components/Album/Item.tsx

```diff
@@ -1,45 +1,44 @@
+import { navigate } from "gatsby"
 import React from "react"
 
-import { deleteFromDynamo } from "../PhotoAnalysis/utils"
 import { ImageRecord } from "./Album"
+import "./Item.scss"
 import { useGetFilePath } from "./useGetFilePath"
 
 interface ItemProps {
   imageRecord: ImageRecord
   dynamoDbItems: ImageRecord[]
-  setDynamoDBItems: React.Dispatch<React.SetStateAction<ImageRecord[]>>
 }
 
 const Item: React.FC<ItemProps> = props => {
   const item = props.imageRecord
-  const { dynamoDbItems, setDynamoDBItems } = props
 
-  const { filepath } = item
-
-  const deleteItem = () => {
-    deleteFromDynamo(item.id)
-    const updatedItemArray = dynamoDbItems.filter(dynamoDbItem => {
-      return dynamoDbItem.id != item.id
-    })
-    setDynamoDBItems(updatedItemArray)
+  const accessibleOnClick = (e: any) => {
```

Comment:
hack to allow clickable images. You can read more about it here: https://dev.to/shivampawar/error-non-interactive-elements-should-not-be-assigned-mouse-or-keyboard-event-jsx-a11yno-noninteractive-elements-interactions-2i1g